### DISCUSSION TITLE
Legacy Windows Supoort (WinXP, Win7)

### DISCUSSION THREAD

**Discussion by hiwooiot** [iterativ/openopc2#49] [General]
Many opcda services are run on winxp systems, can openopc2 run on winxp when python3.8 is based

**Comment by renzop** [comment]
:-)  The last update for Windows XP was in 2008, so thats 15 years ago. We will test Openopc2 on Windows versions that are supported by Microsoft (Windows 10 and 11) only. We strongly believe that the automation industry must move to processes and concepts where security is always part of design decisions (which includes OS updates). 

You can try to run the Openopc2 gateway on Windows XP, would be interesting to know if it runs. Please share your experience here in this discussion. I know that it did runs in Windows 7. But still will not test with theses OSses and therefore not support it.

**Comment by duduyoyo** [comment]
Check this [link](https://github.com/duduyoyo/WebSocket4OPC/discussions/21) out, which provides a feasible way to retrieve OPC data from Windows XP in Python. Hope it helps!
